examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,192 @@
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            png_path,
+            PngFormat,
+            TextureMetadata::srgb_scale(),
+            (),
+            &texture_storage,
+        )
+    };
+    let mut material_texture_set = world.write_resource::<MaterialTextureSet>();
+    let texture_id = material_texture_set.len() as u64;
+    material_texture_set.insert(texture_id, texture_handle);
+
+    let loader = world.read_resource::<Loader>();
+    let sprite_sheet_store = world.read_resource::<AssetStorage<SpriteSheet>>();
+    loader.load(
+        ron_path,
+        SpriteSheetFormat,
+        texture_id,
+        (),
+        &sprite_sheet_store,
+    )
+}
+
+// Initialize a background
+fn init_background_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.z = -10.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world.create_entity().with(transform).with(sprite).build()
+}
+
+// Initialize a sprite as a reference point at a fixed location
+fn init_reference_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 100.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(sprite)
+        .with(Transparent)
+        .build()
+}
+
+fn init_player(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 0.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 1,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(Player)
+        .with(sprite)
+        .with(Transparent)
+        .build()
+}
+
+fn init_camera(world: &mut World, parent: Entity) {
+    let mut transform = Transform::default();
+    transform.translation.z = 1.0;
+    world
+        .create_entity()
+        .with(Camera::from(Projection::orthographic(
+            -250.0, 250.0, 250.0, -250.0,
+        ))).with(Parent { entity: parent })
+        .with(transform)
+        .build();
+}
+
+struct Example;
+
+impl<'a, 'b> SimpleState<'a, 'b> for Example {
+    fn on_start(&mut self, data: StateData<GameData>) {
+        let world = data.world;
+        let resources_path = format!(
```

Comment:
This should really just set the asset directory when creating Application, and then using relative names instead.

Reply:
Didn't realize that field was setting the Asset directory, that would definitely be the right way to do it.
I can change it tonight.

Reply:
Changed